ext/openssl/tests/openssl_cms_decrypt_basic.phpt

```diff
@@ -0,0 +1,66 @@
+--TEST--
+openssl_cms_decrypt() tests
+--SKIPIF--
+<?php if (!extension_loaded("openssl")) print "skip"; ?>
+--FILE--
+<?php
+$infile = __DIR__ . "/cert.crt";
+$privkey = "file://" . __DIR__ . "/private_rsa_1024.key";
+$encrypted = tempnam(sys_get_temp_dir(), "ssl");
+if ($encrypted === false)
+    die("failed to get a temporary filename!");
+$outfile = tempnam(sys_get_temp_dir(), "ssl");
+if ($outfile === false) {
+    unlink($outfile);
+    die("failed to get a temporary filename!");
+}
+
+$single_cert = "file://" . __DIR__ . "/cert.crt";
+$headers = array("test@test", "testing openssl_cms_encrypt()");
+$wrong = "wrong";
+$empty = "";
+
+openssl_cms_encrypt($infile, $encrypted, $single_cert, $headers);
```

Comment:
It would be good to have an encrypt and the decrypt test which would verify that the encrypted envelope can be decrypted.

Reply:
I have to agree.  Let me see what I can do.

Reply:
@Bukka Just to be clear on what we mean by envelope, in this case, can you be explicit?

Reply:
I meant just the enveloped message (result of cms encrypt). It means decrypting the result of cms encrypt and checking that it's the original message...

Reply:
Ok. I've added content into openssl_cms_decrypt_basic.phpt to compare encrypted and decrypted text.